py/selenium/webdriver/common/options.py

```diff
@@ -514,6 +515,15 @@ def add_argument(self, argument) -> None:
     def ignore_local_proxy_environment_variables(self) -> None:
         """By calling this you will ignore HTTP_PROXY and HTTPS_PROXY from
         being picked up and used."""
+        warnings.warn(
+            "using ignore_local_proxy_environment_variables in Options has been deprecated, "
+            "instead, create a Proxy instance with ProxyType.DIRECT to ignore proxy settings, "
+            "pass the proxy instance into a ClientConfig constructor, "
```

Comment:
into the ClientConfig constructor,

Reply:
This is another way to set via Options, so I think should keep this warn separately
I also added another warn in get_remote_connection() if any setting via keyword argument